docs/tracing.md

```diff
@@ -0,0 +1,54 @@
+# Tracing
+
+Distributed tracing is a method of profiling and monitoring applications.  It can provide valuable insight when optimizing and debugging an application.  Mainflux includes the Jaeger open tracing framework as a service with its stack by default.
```

Comment:
Can you please just link a Jeager to official webister ?

Reply:
Good idea.